src/pages/statistics/statistics.css

```diff
@@ -7,10 +7,21 @@
     justify-content: center;
     height: 100vh;
     overflow: hidden;
+    color: black;
 }
 
 .sta {
     font-size: 3.2em;
     line-height: 1.1;
-    color: aquamarine;
+    color: black;
+}
+
+.background {
+    background-color: white;
+    min-height: 100vh;
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    overflow: hidden;
```

Comment:
```suggestion
    background-color: white;
    overflow: auto;
```

Reply:
This is mainly to update the CSS to remove duplicated styling and to change `overflow` into `"auto"`.